app/components/Products.js

```diff
@@ -1,36 +1,53 @@
 import React from 'react';
 import Product from './Product.js';
 import PageSelect from './PageSelect.js';
+import { connect } from 'react-redux';
+import { fetchProducts } from '../redux/thunks/ProductThunks.js';
+import Form from 'react-bootstrap/form';
 
-const Products = ({ products }) => {
-  const PRODUCTS_PER_PAGE = 10;
-  return (
-    <div>
-      <div className="product-search-and-pagination">
-        <div className="filter-and-sort">
-          <select id="filter-by">
-            <option value="">Filter By</option>
-            <option value="option 1">Option 1</option>
-            <option value="option 2">Option 2</option>
-          </select>
-          <select id="sort-by">
-            <option value="">Sort By</option>
-            <option value="option 1">Option 1</option>
-            <option value="option 2">Option 2</option>
-          </select>
+class Products extends React.Component {
+  componentDidMount() {
+    console.log(this.props);
```

Comment:
not sure if we need this console.log anymore

Reply:
you're right!